search-service/src/main/java/de/cxp/ocs/elasticsearch/Searcher.java

```diff
@@ -182,6 +188,7 @@ public SearchResult find(InternalSearchParams parameters) throws IOException {
 		Sample findTimerSample = Timer.start(Clock.SYSTEM);
 		Iterator<ESQueryFactory> stagedQueryBuilders;
 		List<QueryStringTerm> searchWords;
+		//TODO: List<SomeFilterElements> querqyFilters;
```

Comment:
I guess this is just a left-over. The Searcher-Instance should never keep any reference to a request-scoped variable.

Reply:
yup